runtime/j9vm/jvm.c

```diff
@@ -5515,7 +5392,8 @@ JVM_InitClassName
 
 	Trc_SC_GetClassName_Entry(env, theClass);
 
-	if (J2SE_SHAPE_RAW == J2SE_SHAPE(vm)) {
+#ifdef J9VM_IVE_RAW_BUILD /* J9VM_IVE_RAW_BUILD is not enabled */
```

Comment:
J9VM_IVE_RAW_BUILD is not enabled *by default* 
is better

Reply:
sure.